modules/metabat2/metabat2/main.nf

```diff
@@ -33,8 +36,19 @@ process METABAT2_METABAT2 {
 
     mv metabat2/${prefix} ${prefix}.tsv
     mv metabat2 bins
-    bgzip --threads $task.cpus ${prefix}.tsv
-    bgzip --threads $task.cpus bins/*.fa
+
+    gzip ${prefix}.tsv
+    gzip bins/*.fa
+
+    ## Some bash trickery to successfully execute `mv`, even when no contigs are
+    ## actually discarded.
+    ## Thanks to Stéphane Chazelas https://unix.stackexchange.com/a/522283/478356
+    (
+        shopt -s nullglob
+        shopt -u failglob
+        files=(bins/*{lowDepth,tooShort,unbinned}.fa.gz)
+        ((\${#files[@]} == 0)) || mv -- "\${files[@]}" .
+    )
```

Comment:
What about using `find` ?

Something like:

```bash
find ./bins/ -type f -name "*{lowDepth,tooShort,unbinned}.fa.gz" -exec mv {} . \;
```

Reply:
Didn't work when I tried it. When none of the files do not exist, it fails.

Reply:
Ok now it works... :confused:

Reply:
Fix: needed sqaure brackets not curly